### ISSUE TITLE
Fails to read .rev file with sass sub folders

### ISSUE THREAD

**Issue by zhouzi** [Zhouzi/gulp-revise#3]
Gulp task:

```
gulp.task('styles', function() {
  return gulp
    .src('./src/**/*.scss')
    .pipe(sass())
    .pipe(revise())
    .pipe(revise.write('dist'))
    .pipe(gulp.dest('dist'));
});
```

Folder structure:

```
dist/
  pages/
    home_123.css
    home.css.rev
  styles.css
  styles.css.rev
src/
  pages/
    home.scss
  styles.scss
```

Sass creates a corresponding `.css` file for each `.scss` file that lacks a `_` prefix. So it may create some subfolders. The issue is that `revise.write('dist')` consider `dist` to be the output directory (while it's actually `dist/pages` for `home.css`).

It means that `revise.write` fails to read the `.rev` file and thus cannot delete the old revision.

**Comment by zhouzi** [comment]
Fixed as of v1.1.0

**Comment by zhouzi** [comment]
v1.1.0 actually introduced a regression and did not fix this issue. Reopening as it needs more work.

**Comment by zhouzi** [comment]
This issue is now fixed and thoroughly tested as of v1.1.1.
